opentech/apply/funds/templates/funds/includes/review_table.html

```diff
@@ -1,33 +1,56 @@
 {% load review_tags %}
 <table class="reviews-sidebar js-reviews-sidebar">
-    {% if staff_reviews or object.staff_not_reviewed %}
-        {% if staff_reviews %}
-            <tr class="tr tr--subchild light-grey-bg">
-                <th colspan="2"></th>
-                <th>{{ object.reviews.submitted.recommendation|traffic_light }}</th>
-                <th>{{ object.reviews.submitted.get_score_display|default_if_none:'-' }}</th>
-            </tr>
-            {% include 'funds/includes/review_table_row.html' with reviews=staff_reviews %}
-        {% endif %}
+    {% if top_reviews %}
+        <tr class="tr tr--subchild light-grey-bg">
+            <th colspan="2"></th>
+            <th>{{ recommendation|traffic_light }}</th>
+            <th></th>
+        </tr>
+    {% endif %}
 
-        {% if object.staff_not_reviewed %}
-            {% include 'funds/includes/review_table_row.html' with reviews=object.staff_not_reviewed missing=True %}
-        {% endif %}
-    {% else %}
+    {% if reviews_block.role_reviewed or reviews_block.role_not_reviewed %}
+        {% for review_data in reviews_block.role_reviewed %}
+            {% include 'funds/includes/review_table_row.html' with review=review_data.review reviewer=review_data.reviewer role=review_data.role %}
+        {% endfor %}
+
+        {% for review_data in reviews_block.role_not_reviewed %}
+            {% include 'funds/includes/review_table_row.html' with reviewer=review_data.reviewer role=review_data.role missing=True %}
+        {% endfor %}
+
+    {% endif %}
+
+    {% if reviews_block.staff_reviewed or reviews_block.staff_not_reviewed %}
+        {% for review_data in reviews_block.staff_reviewed %}
+            {% include 'funds/includes/review_table_row.html' with review=review_data.review reviewer=review_data.reviewer %}
+        {% endfor %}
+
+        {% for review_data in reviews_block.staff_not_reviewed %}
+            {% include 'funds/includes/review_table_row.html' with reviewer=review_data.reviewer missing=True %}
+        {% endfor %}
+
+    {% endif %}
+
+    {% if not top_reviews %}
```

Comment:
This is not correct - we only want to display this message if there is nothing on the review. no staff, no roles, no nothing. 

```
{% if not reviews_block.role_reviewed 
or not reviews_block.role_not_reviewed 
or not reviews_block.staff_reviewed 
or not reviews_block.staff_not_reviewed %}
```
Though maybe neater...

Reply:
Ok, I didn't know that requirement. I've adjusted now!